### ISSUE TITLE
Test E-mail: Recipient email addresses do not validate

### ISSUE THREAD

**Issue by johnj24** [barrelstrength/craft-sprout-email#162]
### Description (screenshots, logs, ...)
I'm trying to send a test e-mail and I've put my personal Gmail address into the provided Recipients box in the pop-up modal. Upon hitting the "Send Test" button, I'm given this error:
```
Error
Recipient email addresses do not validate:
```
I've triple checked my email is correct. I don't know if this is a new bug or not because I've had success sending test e-mails before.

### Steps to reproduce

1. Go to Sprout Email > Notifications
2. Click "Send Test" button next to an email notification
3. Put in a valid e-mail address
4. Hit "Send Test" button

### Additional info

- Craft version: 3.4.17.1
- PHP version: 7.0.33
- Database driver & version:  MySQL 5.7.30
- Plugins & versions: Sprout Email 4.4.3

_For some issues, we may request that you send the following items to Support <sprout@barrelstrengthdesign.com>. If you feel your issue may fall in that category, feel free to send things over after you post this issue and reference the issue in your email._

- `composer.json`
- `storage/logs/web.log` (or a zip of the whole log folder)
- An export of your database (or, if you can't share the whole db, the specific Sprout tables)

**Comment by BenParizek** [comment]
@johnj24 In my tests this appears to be working properly, however, I did discover another bug in my testing. It also appears that when sending a test email, the List and Other Recipient settings may not be excluded properly. So, the error your seeing could relate to some of those other recipient settings field in your Notification Email configuration.

Let me know if that helps track things down. I'll work on getting a fix in place so a test email is limited to just the email you put in that box. And we can circle back here if you're still seeing the error.

**Comment by johnj24** [comment]
@BenParizek I'm not seeing anything out of place in my notification settings. I've set up a mailing list in Sprout Reports with an e-mail column. I've then used that mailing list in the e-mail notification. No addresses are in the `To:` line either.

Here are some screenshots
![Screenshot 2020-05-20 15 03 21](https://user-images.githubusercontent.com/1458917/82492412-c2697e00-9aab-11ea-9e32-6e99746fc642.png)
![InkedScreenshot 2020-05-20 15 03 34_LI](https://user-images.githubusercontent.com/1458917/82492425-c7c6c880-9aab-11ea-99e5-ab26e215d6e0.jpg)
